packages/webpack/src/config/base.js

```diff
@@ -397,6 +397,15 @@ export default class WebpackBaseConfig {
     }
 
     // Clone deep avoid leaking config between Client and Server
-    return this.extendConfig(cloneDeep(config))
+    const extendedConfig = this.extendConfig(cloneDeep(config))
+    const optimization = extendedConfig.optimization
```

Comment:
it can, the thing is that in our docs we specifically provide example where its set in extend
https://nuxtjs.org/api/configuration-build#extend
And breaking it somehow a breaking change for some users. Thats first point. Second point is that it might be set in extend only for client like in example from link, while nuxt.config dont allow such option

Reply:
Oh, ok.
BTW, optimization can be false